config/locales/es-ES/general.yml

```diff
@@ -45,12 +44,10 @@ es:
       responses:
         one: 1 Respuesta
         other: "%{count} Respuestas"
-        zero: Sin respuestas
```

Comment:
@bertocq Commonly we follow CLDR and English has only 2 forms: one and other. The zero form is trimmed on import and won't be included into exported files. Zero belongs to "other" plural form too, so your app will show "0 Respuestas" in Spanish. Anyway, checking with team how we can improve the current behavior